.travis.yml

```diff
@@ -306,6 +310,12 @@ before_install:
         # this assumes TEST_THEANO runs in the same build as TEST_MATPLOTLIB
         conda install theano;
     fi
+
+  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
+        conda config --add channels conda-forge;
+        conda config --add channels symengine;
```

Comment:
Also, why don't we release a symengine, so that sympy master always depends on a released version of symengine (i.e. with a git tag).